content/enterprise/operational-modes/index.mdx

```diff
@@ -0,0 +1,169 @@
+
+Terraform Enterprise data utilizes the following types of data:
+
+- **PostgreSQL Database** - Stateful Terraform Enterprise application data. (e.g. workspace settings, organization settings, run information, user information)
+
+- **Object Storage** - Artifacts generated by Terraform Enterprise. (e.g. state files, plan files, run logs, configuration versions)
+
+- **Vault** - Encryption keys used to encrypt and decrypt objects within object storage.
+
+- **Configuration** - Configuration settings. (e.g. the hostname, object storage credentials, database credentials, concurrency settings, etc.)
+
+## Operational Modes
+
+The table below lists each operational mode and where it stores each type of data.
+
+| Operational Mode                        | PostgreSQL Database                                                              | Object Storage                                                                  | Vault                                                                             | Configuration                              |
+| --------------------------------------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- | --------------------------------------------------------------------------------- | ------------------------------------------ |
+| [Demo](#demo)                           | Stored in a Docker volume on the instance.                                       | Stored in a Docker volume on the instance.                                      | Stored in the PostgreSQL Database unless using [External Vault](#external-vault). | Stored in a Docker volume on the instance. |
+| [Mounted Disk](#mounted-disk)           | Stored in a directory on the instance backed by user-managed persistent storage. | Stored in a directory on the instance backed by user-managed persisent storage. | Stored in the PostgreSQL Database unless using [External Vault](#external-vault). | Stored in a Docker volume on the instance. |
+| [External Services](#external-services) | Stored in an external user-managed PostgreSQL database.                          | Stored in an external user-managed object storage location.                     | Stored in the PostgreSQL Database unless using [External Vault](#external-vault). | Stored in a Docker volume on the instance. |
+
+## Demo
+
+!> **Warning:** Do not use the Demo operational mode in a production environment. Terraform Enterprise data is lost when the instance terminates.
+
+The Demo operational mode stores all data (PostgreSQL Database, Object Storage, Configuration) in Docker volumes on the instance. This allows you to quickly deploy Terraform Enterprise without any additional infrastructure. However, when the instance terminates, all data is lost.
+
+The Demo operational mode is best when you:
+
+- Are okay with all the data being lost when the instance terminates.
```

Comment:
```suggestion
- Understand that all data (PostgreSQL Database, Object Storage, Configuration) will be lost when the instance terminates.
```
It might be best to fully clarify, again, what is at stake if they choose demo mode.

Reply:
+1. I actually even wonder if we should change this slightly and then put this callout in a callout for emphasis. So like this bullet would say "Do not need to keep the data permanently" and then provide an example of when that would be. And then have a separate callout somewhere like:

~> **Note:** All data will be lost when the instance terminates." 

Maybe too much, but I want to make sure people don't use this unless it's really appropriate.

Reply:
The demo operational mode is gone now so I get to do the best choice and remove this altogether!